openssh/files/sshd_config

```diff
@@ -69,7 +68,7 @@
 
 # Authentication:
 {{ option_default_uncommented('LoginGraceTime', 120) }}
-{{ option_default_uncommented('PermitRootLogin', 'no') }}
+{{ option_default_uncommented('PermitRootLogin', 'yes') }}
```

Comment:
Please update the readme clarifying that people need to explicitly turn off root login, looks ok to merge.